include/labwc.h

```diff
@@ -318,6 +319,10 @@ struct server {
 	struct wlr_pointer_constraints_v1 *constraints;
 	struct wl_listener new_constraint;
 
+	struct wlr_tearing_control_manager_v1 *tearing_control;
+	struct wl_listener tearing_new_object;
+	struct wl_list tearing_controllers;
```

Comment:
This seems unused...

Reply:
Oh yea wl_list tearing_controllers is unused, i tried if it works without that part and it did so i removed stuff and forgot that. Other 2 are used in server.c.